ibctest/test_setup.go

```diff
@@ -39,15 +31,15 @@ const (
 type IBCTestCase struct{}
 
 // uses reflection to get test case
-func GetTestCase(testCase string) (func(testName string, cf ChainFactory, relayerImplementation RelayerImplementation) error, error) {
+func GetTestCase(testCase string) (func(testName string, cf ChainFactory, relayerImplementation ibc.RelayerImplementation) error, error) {
```

Comment:
I think this function is unused now; I should have deleted it in a previous change.